doc/man3/OSSL_trace_set_channel.pod

```diff
@@ -214,8 +214,6 @@ the following:
      BIO_dump(trc_out, bar, sizeof(bar));
  } OSSL_TRACE_END(TLS);
 
-=head1 Simple example
-
```

Comment:
It should remain as a head2

Reply:
okay, pushed.

(For what it's worth, I think we split things up with head2 way too often but that's just a side-note).

Reply:
I disagree with you. The sub-sections makes it look less like walls of text.

Reply:
This isn't Moby Dick or Finnegan's Wake; it's a manpage and there's under 82,000 lines split over nearly 600 files.  Shrug.